exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
+let message = "This is a string"
+//log
 console.log(message);
```

Comment:
Below this line, you were supposed to console the data type of the variable message using typeof operator.

Reply:
Thanks, Anthony